blueprints/secops/sops/ops2deb.yml

```diff
@@ -0,0 +1,18 @@
+name: sops
+matrix:
+  versions:
+    - 3.8.0
+    - 3.7.3
+    - 3.7.2
+  architectures:
+    - amd64
+    - arm64
+homepage: https://age-encryption.org/
+summary: Simple and flexible tool for managing secrets
```

Comment:
Somewhere in the packaging guidelines it is mentioned that the summary should start by a lowercase if possible

Reply:
thx fixed